doc/developers/contributing.rst

```diff
@@ -44,6 +44,16 @@ ticket to the
 <https://github.com/scikit-learn/scikit-learn/issues>`_. You are also
 welcome to post feature requests or pull requests.
 
+Automated Contributions Policy
+==============================
+
+Please refrain from submitting mass issues or pull requests generated by
```

Comment:
I think we don't want any issues that are auto-generated (unless it is a collaboration).
```suggestion
Please refrain from submitting issues or pull requests generated by
```